src/handlers/motions/motionFinalized/helpers.ts

```diff
@@ -353,3 +358,77 @@ export const updateColonyUnclaimedStakes = async (
     });
   }
 };
+
+/*
+ * When a Simple Payment motion is finalized, we want to update the action in the database
+ * to include the networkFee and the amount excluding the fee at the time of finalization
+ */
+export const updateAmountToExcludeNetworkFee = async (
+  action: string,
+  colonyAddress: string,
+  finalizedMotion: ColonyMotion,
+): Promise<void> => {
+  const colonyClient = await getCachedColonyClient(colonyAddress);
+  const oneTxPaymentClient =
+    (await colonyClient?.getExtensionClient(Extension.OneTxPayment)) ?? null;
+
+  const parsedAction = parseAction(action, {
+    colonyClient,
+    oneTxPaymentClient,
+  });
+
+  if (!parsedAction) {
+    return;
+  }
+
+  if (parsedAction.name !== ColonyOperations.MakePaymentFundedFromDomain) {
+    return;
+  }
+
+  const { data } =
+    (await query<
+      GetColonyActionByMotionIdQuery,
+      GetColonyActionByMotionIdQueryVariables
+    >(GetColonyActionByMotionIdDocument, {
+      motionId: finalizedMotion.id,
+    })) ?? {};
+
+  const colonyAction = data?.getColonyActionByMotionId?.items[0];
+
+  if (!colonyAction) {
+    return;
+  }
+
+  if (colonyAction.type === ColonyActionType.PaymentMotion) {
+    if (colonyAction.networkFee) {
+      return;
+    }
+
+    if (!colonyAction.amount) {
+      return;
+    }
+
+    const networkInverseFee = await getNetworkInverseFee();
+    if (!networkInverseFee) {
+      throw new Error(
+        'Network inverse fee not found. This is a bug and should be investigated.',
+      );
+    }
+
+    const amountWithFee = colonyAction.amount;
+
+    const amountLessFee = getAmountLessFee(amountWithFee, networkInverseFee);
```

Comment:
I know it wasn't introduced in this PR, but maybe we could rename this helper function in `getAmountExcludingFee`?